README.md

```diff
@@ -21,22 +21,27 @@ any contributions or feedback for the project.
 
 For a full desktop experience you will also need the following external tools installed:
 
-* xbacklight
-* arandr
+- xbacklight
+- arandr
 
 # Getting Started
 
-Using standard go tools you can install Fyne's desktop using:
+Using standard go tools you can install FyneDesk using:
+```
+go get fyne.io/fynedesk/cmd/fynedesk
+```
 
-    go get fyne.io/fynedesk/cmd/fynedesk
+Once installed you can run the following commands to get everything set up. This will take care of all the building and configuration for you.
+```
+make
```

Comment:
Before this works you need to be in the right directory, which the user will not be if they just executed 'go get'.

Reply:
Good catch. Didn't think of that.